### ISSUE TITLE
no audio from Zoom participants

### ISSUE THREAD

**Issue by skierpage** [flathub/us.zoom.Zoom#366]
I'm using version 5.13.4.711 of the Flatpak on Fedora 37 KDE Spin.
Zoom meeting links open in Firefox then launch the Zoom flatpak.

I see the other participants in the meeting, chat works, but I don't hear any audio. The Zoom  [Join Audio] button appears but clicking it does nothing. There are no error messages in `journalctl`.

The Zoom "Test speakers and microphone" dialog that appears before you join works fine, which suggests the flatpak can access sound. The KDE Skrooge flatpak makes sounds fine.

Other bugs, mostly closed, indicate that Zoom depends on pulseaudio. Fedora 37 uses Pipewire which provides the pulseaudio API. I started a --devel instance of zoom to try to debug this; I ran a plausible `/app/extra/zoom/ZoomLauncher` command I saw in `ps` output; this started Zoom and I was able to join the meeting, but still no sound from meeting participants. There were  no console errors. In this --devel instance I typed `paplay /app/extra/zoom/sip/Hangup.WAV` and it made the sound. I also ran `pactl info`
```
  [📦 us.zoom.Zoom ~]$ pactl info
        Server String: unix:/run/flatpak/pulse/native
        Library Protocol Version: 35
        Server Protocol Version: 35
        Is Local: yes
        Client Index: 390
        Tile Size: 65472
        User Name: spage
        Host Name: fedlaptop
        Server Name: PulseAudio (on PipeWire 0.3.64)
        Server Version: 15.0.0made
        Default Sample Specification: float32le 2ch 48000Hz
        Default Channel Map: front-left,front-right
        Default Sink: alsa_output.pci-0000_00_1f.3.hdmi-stereo
        Default Source: alsa_input.pci-0000_00_1f.3.analog-stereo.5
        Cookie: 1e21:8064
```
so audio seems set up in the flatpak, but whatever Zoom is doing to play meeting audio, it's not working.

What else can I do to debug this?

**Comment by violetmage** [comment]
Of course, you should double check all of your audio related settings in zoom's settings menu. I don't personally use the desktop client, so maybe there is some sort of participent-specific volume adjustment (similar to discord) that got messed up?

Additionally, you could try uninstalling zoom with `flatpak remove --delete-data us.zoom.Zoom`, then reinstalling, to see if starting with a blank configuration helps.

The troubleshooting you have already done makes sense (very nicely explained), and to me confirms that the issue is internal to zoom.
